cabal.project

```diff
@@ -13,3 +13,22 @@ source-repository-package
 package inspector
   -- This package uses bang patterns without enabling them
   ghc-options: -XBangPatterns
+
+-- This is not on Hackage.
+source-repository-package
+  type: git
+  location: https://github.com/primetype/inspector
+  tag: 962c6b5b52b760eeeb847d318800392be9aab5fa
```

Comment:
Shouldn't this be forked into `input-output-hk` organisation and published to CHaP?

Reply:
Its only used in the tests, so we do not need it in CHaP.

Reply:
Please add comment to that effect.

Reply:
Do we have a policy that explicitly says we should not push test dependencies to CHaP?